app/javascript/packages/form-steps/form-steps.tsx

```diff
@@ -246,6 +250,15 @@ function FormSteps({
   const didSubmitWithErrors = useRef(false);
   const forceRender = useForceRender();
   const ifStillMounted = useIfStillMounted();
+  const [previousErrors, setPreviousErrors] = useState<PreviousErrorsLookup>({});
+
+  useEffect(() => {
+    const prevErrs = previousErrors[stepName || steps[0]?.name];
```

Comment:
In at least one other place in the file we reference the step name by `step?.name`, since `step` would be the result of actually validating `stepName` in the set of steps. We could move this `useEffect` a bit down (or conversely, move the `step` assignment a bit up) to use it.

Reply:
Still works as intended, so the `useEffect` has been moved.